### ISSUE TITLE
Current and Total

### ISSUE THREAD

**Issue by jonau01** [Ashfaaq18/OpenNetMeter#55]
Version 0.10 has  : Current Data  and Total Data .    They both seem to show cumulative.
Version 0.11 has :  Current Data  .                             This seems to be a cumulative rather than current.   The number keeps rising.

It is possible to have an Actual Current Data and  Total Data  like the display of version 0.10

By  "Current Data Received"  :  I would like to see the  Actual Data received  since last refresh.    If an application (example chrome)  is receiving 10 KB (since last "refresh")   than it should show 10KB  rather than ( total of all KB received since OpenNetMeter the  is started )
By  "Total Data Received" :  is the total  data received since the OpenNetMeter started ( this is what is shown as Current Data Received for both version 0.10 and 0.11

I mentioned only "Data Received" as an example both it should be both applied to Received and Sent

Example 

```
Name      Current Data Received   Total Data Received        ( 04-Jul-2022 10:47:01 )
Brave     10KB                                 10KB

Name     Current Data Received    Total Data Received        ( 04-Jul-2022 10:47:02 )
Brave     10KB                                 20KB                             

Name     Current Data Received    Total Data Received        (04-Jul-2022  10:47:03 )
Brave     9KB                                  29KB

Name     Current Data Received    Total Data Received        (04-Jul-2022  10:47:04 )
Brave    11KB                                 40KB                                 
```

Thanks

Here is what I  see with OpenNetMeter-0.10.zip

![image](https://user-images.githubusercontent.com/65045445/177183426-f9930dc4-29ff-419d-af57-9742c6f38837.png)

**Comment by Ashfaaq18** [comment]
Hey @jonau01 , thank you for the suggestion. Its actually a better format compared to what i have implemented.

> Example
> 
> ```
> Name      Current Data Received   Total Data Received        ( 04-Jul-2022 10:47:01 )
> Brave     10KB                                 10KB
> 
> Name     Current Data Received    Total Data Received        ( 04-Jul-2022 10:47:02 )
> Brave     10KB                                 20KB                             
> 
> Name     Current Data Received    Total Data Received        (04-Jul-2022  10:47:03 )
> Brave     9KB                                  29KB
> 
> Name     Current Data Received    Total Data Received        (04-Jul-2022  10:47:04 )
> Brave    11KB                                 40KB                                 
> ```
> 

And thank you for the above example. this helped me understand what you meant.
In one of the following releases, ill include, 
a "Current Data received" column showing data received every second (not cumulative) like your example table.
a "Total Data Received" column showing data received (cumulative) like your example table.

**Comment by Ashfaaq18** [comment]
This feature is now available in the v0.12.0 release
